src/browser.ts

```diff
@@ -224,8 +224,13 @@ export class AnalyticsBrowser {
       analytics.page().catch(console.error)
     }
 
-    if (window.location.search.includes('ajs_')) {
-      analytics.queryString(window.location.search).catch(console.error)
+    const search = window.location.search ?? ''
+    const hash = window.location.hash ?? ''
+
+    const term = search.length ? search : hash.replace(/#\//, '')
```

Comment:
is the replacement needed? If so, do we need to cover something like this: `www.test.com/#about?id=123` ?

Reply:
for this example: `http://localhost:3000/#/?ajs_id=123`, the replacement is needed so we have the same format for both search and hash (`?` + `key` + `=` + `val`).

for `www.test.com/#about?id=123` we'd need to remove everything that's after the `#` and before the `?`

I can rework the regex to cover that

Reply:
this `/(?=#).*(?=\?)/` should do the trick